Dockerfile

```diff
@@ -85,6 +85,7 @@ RUN chmod +x /usr/local/bin/doctor && \
     chmod +x /usr/local/bin/ssl && \
     chmod +x /usr/local/bin/test && \
     chmod +x /usr/local/bin/vars && \
+    chmod +x /usr/local/bin/retry && \
```

Comment:
Maybe we can have a prefix for queue related tasks. Not sure how complex `queue retry` will be, but we can also consider `queue-retry`, I would also consider a few command to give the size of the queue, and size of failed jobs.

Maybe

`queue-count-jobs`
`queue-count-failures`

WDYT?